tests/boost/dataset_broker_producer2.cpp

```diff
@@ -22,15 +22,20 @@ using json = nlohmann::json;
 using namespace std::string_literals;
 
 BOOST_FIXTURE_TEST_CASE(_dataset_manager_general, CompareCTypes) {
+    BOOST_CHECK(boost::unit_test::framework::master_test_suite().argc == 2);
+    int broker_port = atoi(boost::unit_test::framework::master_test_suite().argv[1]);
+    BOOST_CHECK(broker_port);
+
     _global_log_level = 5;
     __enable_syslog = 0;
 
     // We have to start the restServer here, because the datasetManager uses it.
```

Comment:
Doesn't the dM only need the client?

Reply:
I figured this out. It's for the `force-update` I think.

Reply:
Correct. I'm adding this to the comment.